code/src/App.js

```diff
@@ -1,9 +1,114 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Well done for managing so many components! 
Perhaps also include a clue to what the question is about in the name of the question, to make it easier if the questions would change order or if more people would work in the same project.